src/components/classifier/SwipeCard.js

```diff
@@ -99,64 +120,116 @@ class SwipeCard extends Component {
             inputRange: [-windowWidth/4, 0, windowWidth/4],
             outputRange: [1, 0, 1]
           })
-
-        const imageDimensionStyle = {
-            width: this.state.overlayViewWidth,
-            height: this.state.overlayViewHeight
-          }
-        const alreadySeen = (this.props.subject.already_seen || this.props.seenThisSession) && this.state.imageIsVisible
+        const alreadySeen = (this.props.subject.already_seen || this.props.seenThisSession)
     
         const overlay =
-            <View>
-                <Animated.View style={[imageDimensionStyle, styles.overlayContainer, styles.overlayBackground, { opacity }]}>
-                    <FontedText style={styles.overlayText}>
-                        { this.props.answers[this.state.overlayAnswerIndex].label }
-                    </FontedText>
-                </Animated.View >
-            </View>
+            <Animated.View style={[styles.overlayContainer, styles.overlayBackground, { opacity }]}>
+                <FontedText style={styles.overlayText}>
+                    { this.props.answers[this.state.overlayAnswerIndex].label }
+                </FontedText>lop
```

Comment:
Should this "lop" be here after `</FontedText>`?

Reply:
Woops! That shouldn't be there